microagent/mf2c-entrypoint.sh

```diff
@@ -70,9 +70,11 @@ docker run -d --network="host" \
 docker run -d --hostname=IRILD039 --privileged \
         -e LEADER_ENDPOINT="https://dashboard.mf2c-project.eu" \
         -v /var/run/docker.sock:/var/run/docker.sock \
+        -v /etc/hostname:/etc/hostname
```

Comment:
why are you sharing the hostname and then also setting the container hostname with `--hostname`?